server/index.ts

```
@@ -0,0 +1,690 @@
+};
+
+// Helper function to convert a ProseMirror node to HTML
+const convertNodeToHtml = (node: any): string => {
+  if (!node || typeof node !== 'object') return '';
+  
+  switch (node.type) {
+    case 'paragraph':
+      const content = node.content ? node.content.map((child: any) => convertNodeToHtml(child)).join('') : '';
+      return `<p>${content}</p>`;
+    
+    case 'text':
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)